SwiftProtobuf.xcodeproj/project.pbxproj

```diff
@@ -2193,6 +2205,9 @@
 				PRODUCT_NAME = SwiftProtobuf;
 				SDKROOT = watchos;
 				TARGETED_DEVICE_FAMILY = 4;
+				VALID_ARCHS = "";
+				"VALID_ARCHS[sdk=watchos*]" = "arm64_32 armv7k";
+				"VALID_ARCHS[sdk=watchsimulator*]" = "x86_64 i386";
```

Comment:
What happens in older Xcodes where arm64_32 isn't valid?  Since we try to support a full Swift version back (https://github.com/apple/swift-protobuf#system-requirements), that still means older Xcodes.

Reply:
That's also a great question. This doesn't really future-proof it either...

Reply:
It seems that you can add gibberish here and xcode will still build fine as long as there's a valid architecture.